posneg.c

```diff
@@ -0,0 +1,20 @@
+/* Yael Kelmer.
```

Comment:
Perfect job! The only thing I'd add is maybe reconsider how you handle the 0! It wasn't a requirement, but it's something to think about for next time! 😃